config/channels/in-memory-channel/300-in-memory-channel.yaml

```diff
@@ -38,18 +38,18 @@ spec:
         # see issue: https://github.com/knative/serving/issues/912
         x-kubernetes-preserve-unknown-fields: true
     additionalPrinterColumns:
-    - name: Ready
```

Comment:
@navidshaikh  what's the motivation for this kind of re-grouping ?

Reply:
@matzew @pierDipi : 
- The value for `REASON` could be empty if resource is READY, thus printing an empty column in between non empty columns
- If resource is not `READY`, we can add `reason:message` for `REASON` column and keeping the variable length cell at the right (easy to parse)
- To keep it consistent with how serving resources' fields are grouped

Reply:
thanks for the feedback, @navidshaikh